src/components/OrderBuySell/index.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import styled from 'styled-components'
+import Tabs from 'components/common/Tabs/Tabs'
+
+export interface TabItemType {
+  readonly id: number
+  title: string
+  content: string
+  readonly activeColor?: string
+}
+
+// pass string props to indicate what content/component to show
+
+const tabItems: TabItemType[] = [
+  {
+    id: 1,
+    title: 'BUY',
+    content: '- buy component -',
+    activeColor: '--color-buy',
+  },
+  {
+    id: 2,
+    title: 'SELL',
+    content: '- sell component -',
+    activeColor: '--color-sell',
```

Comment:
not sure if we're keeping `var(--color-style-thing)` variables in CSS, @Velenir @anxolin any input? My understanding was that we move to JS style CSS and theming

Reply:
Yeah probably better, as I saw in your Button component. So in this case we could use a JS theme variable like `sellColor` or something. Perhaps for a future PR?

Reply:
yeah later is fine totally